### ISSUE TITLE
Outline View dark mode color polish

### ISSUE THREAD

**Issue by violasong** [devtools-html/debugger.html#7426]
These two instances of blue should be replaced by the deeper blue color used in the other search focus outlines, tab highlight, etc.
<img width="310" alt="image" src="https://user-images.githubusercontent.com/55994/49522519-d7c9f880-f875-11e8-81cf-7018d8affa26.png">

**Comment by notjaril** [comment]
I can claim this :)

**Comment by AnshulMalik** [comment]
Sure @notjaril It's all yours :)

**Comment by notjaril** [comment]
@violasong Just wanted to clarify. You mentioned the focus outlines and tab highlight as the "deeper blue color", but it seems both of those are different colors in the first place.

Collected the current colors being used for reference:

Element | CSS Var | Light Theme | Dark Theme |
| --- | --- | --- | --- |
Outline Filter |--theme-highlight-blue | #0074e8 | #0a84ff |
Sort By Name Button | --theme-highlight-blue | #0074e8 | #0a84ff |
Search Focus Outline | --theme-selection-background | #0074e8 | #204e8a |
Tab Highlight | --tab-line-selected-color | #0a84ff | #0a84ff |

<img width="400" alt="table" src="https://user-images.githubusercontent.com/15959269/49541548-eb7e5a80-f887-11e8-9cf4-1ab8f1f71bfc.png">

Based on this, I take it you're asking to change the --theme-highlight-blue for the Dark Theme to be changed to the same color as Search Focus Outline instead (#204e8a)?

**Comment by violasong** [comment]
Thanks for this investigation - by search focus outline, I meant the 1px wide border around the input field (which is #0084FF like the tab highlight). So yes, it would be great if you could change --theme-highlight-blue to --tab-line-selected-color. (I think the other color will be too dark here.)

**Comment by violasong** [comment]
Oh wait, I take that back - theme-highlight-blue is being used everywhere as the main syntax highlighting color in dark mode, so it shouldn't be changed from its light-colored blue. You probably need some other variable for these to change them to #0a84ff (It's also called Blue 50 in the code)

**Comment by violasong** [comment]
Ah, and sorry to make this more complicated, but I realized the Sort by Name icon should indeed be that darker clolor (#204e8a) so that it matches the toggle buttons in network/console.

**Comment by violasong** [comment]
Yes, this looks great! Thank you @notjaril!
